src/eva/core/models/modules/typings.py

```diff
@@ -21,3 +21,16 @@ class INPUT_BATCH(NamedTuple):
 
     metadata: Dict[str, Any] | None = None
     """The associated metadata."""
+
+
+class INPUT_TENSOR_BATCH(NamedTuple):
```

Comment:
do we need to separate classes `INPUT_BATCH` and `INPUT_TENSOR_BATCH`, the only difference being that `INPUT_TENSOR_BATCH`targets can't be None?

Reply:
yes indeed, this is mainly used in batch writter, maybe I can move it there 🤔